src/main/java/com/sda/calc/Calculator.java

```diff
@@ -0,0 +1,24 @@
+package com.sda.calc;
+
+public class Calculator<i> {
```

Comment:
A co tutaj robi to `<i>`? :) Wygląda jak parametr typu generycznego, ale napisany małą literą i nieużywany w klasie. :)